__tests__/app.test.js

```diff
@@ -148,4 +147,67 @@ describe('GET /api/articles/:article_id/comments task 6',() => {
             expect(res.body.comments).toEqual([])
         });
     });
+});
+
+//8
+describe('PATCH /api/articles/:article_id task 8',() => {
+    test('responds with status code 200, updates article and returns updated article',() => {
+        const reqArticleID = 1;
+        const votesToUpdate = {inc_votes: 29}
+        return request(app).patch(`/api/articles/${reqArticleID}`)
+        .send(votesToUpdate)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.updatedArticle).toEqual(
+                expect.objectContaining({
+                    article_id: 1,
+                    title: "Living in the shadow of a great man",
+                    topic: "mitch",
+                    author: "butter_bridge",
+                    body: "I find this existence challenging",
+                    created_at: expect.any(String),
+                    votes: 129,
```

Comment:
Great test, love the assertion that the votes have increased by the correct amount